src/core/client/admin/routes/Configure/sections/Moderation/PreModerateAllCommentsConfig.tsx

```diff
@@ -0,0 +1,142 @@
+  });
+
+  const { input: moderationInput } = useField<string>("moderation");
+
+  const onAddSite = useCallback(
+    (siteID: string) => {
+      const changed = [...premoderateAllCommentsSitesInput.value];
+      const index = changed.indexOf(siteID);
+      if (index === -1) {
+        changed.push(siteID);
+      }
+      premoderateAllCommentsSitesInput.onChange(changed);
+    },
+    [premoderateAllCommentsSitesInput]
+  );
+
+  const onRemoveSite = useCallback(
+    (siteID: string) => {
+      const changed = [...premoderateAllCommentsSitesInput.value];
+      const index = changed.indexOf(siteID);
+      if (index >= 0) {
+        changed.splice(index, 1);
+      }
+      premoderateAllCommentsSitesInput.onChange(changed);
+    },
+    [premoderateAllCommentsSitesInput]
+  );
+
+  return (
+    <>
+      <Field name="moderation" type="radio" value="PRE">
+        {({ input }) => (
+          <RadioButton {...input} id={`${input.name}-PRE`} disabled={disabled}>
+            <Localized id="configure-moderation-allSites">
+              <span>All sites</span>
+            </Localized>
+          </RadioButton>
+        )}
+      </Field>
+      <Field name="moderation" type="radio" value="SINGLE_SITES_PRE">
+        {({ input }) => (
+          <>
+            <RadioButton
+              {...input}
+              id={`${input.name}-SINGLE_SITES_PRE`}
+              disabled={disabled}
+            >
+              <Localized id="configure-moderation-singleSites">
+                <span>Single sites</span>
+              </Localized>
+            </RadioButton>
+          </>
+        )}
+      </Field>
+      {moderationInput.value === GQLMODERATION_MODE.SINGLE_SITES_PRE && (
+        <div className={styles.singleSites}>
+          <HorizontalGutter spacing={3} mt={3} mb={3}>
+            {premoderateAllCommentsSitesInput.value.map((siteID: string) => {
+              return (
+                <PreModerationSitesSelectedQuery
+                  key={siteID}
+                  siteID={siteID}
+                  onChange={onRemoveSite}
+                />
+              );
+            })}
+          </HorizontalGutter>
+          <SiteSearch
+            showOnlyScopedSitesInSearchResults
+            showAllSitesSearchFilterOption={false}
+            showSiteSearchLabel={false}
+            onSelect={onAddSite}
+          />
+          {hasError(premoderateAllCommentsSitesMeta) ? (
+            <Localized id="configure-moderation-singleSites-validation">
+              <ValidationMessage className={styles.validationMessage}>
+                You must select at least one site.
+              </ValidationMessage>
+            </Localized>
+          ) : null}
+        </div>
+      )}
+      <Field name="moderation" type="radio" value="POST">
+        {({ input }) => (
+          <RadioButton {...input} id={`${input.name}-POST`} disabled={disabled}>
+            <Localized id="configure-onOffField-off">
```

Comment:
Is there any reason not to also use this localization here for the `Off` option, from the on/off field?

Reply:
Nah, this is fine. `configure-onOffField-off` is a generic component localization for the entirety of the configure form.